addons/account/models/account.py

```diff
@@ -705,7 +705,8 @@ def compute_all(self, price_unit, currency=None, quantity=1.0, product=None, par
                 taxes += ret['taxes']
                 continue
 
-            tax_amount = tax._compute_amount(base, price_unit, quantity, product, partner)
+            tax_excluded = tax_included = tax_base = base
```

Comment:
Jep, the only reason behind would be to help other developers who use the result of compute all to clearly understand the values and who knows what it is good for in applied usage...sometimes it is nice and important to have all the values and not just the magic output.